osu.Game/Overlays/Chat/ChatTabControl.cs

```diff
@@ -143,6 +144,13 @@ private void fadeInactive()
                 textBold.FadeOut(transition_length, Easing.OutQuint);
             }
 
+            protected override bool OnMouseUp(InputState state, MouseUpEventArgs args)
+            {
+                if (args.Button == MouseButton.Middle)
+                    closeButton.Action();
+                return true;
```

Comment:
should return true only when button is middle.

Reply:
Oh right. Gonna fix that

Reply:
I was okay with it always returning `true`, because channel tabs should never let input pass through.

Reply:
But here is `OnMouseUp`. Maybe should block `OnMouseDown` additionally in that case.

Reply:
No big issue for me either way.